types/jasmine/index.d.ts

```diff
@@ -1182,30 +1252,58 @@ declare module "jasmine" {
         onCompleteCallbackAdded: boolean;
         defaultReporterConfigured: boolean;
 
-        constructor(options: any);
+        constructor(options: jasmine.JasmineOptions);
         addMatchers(matchers: jasmine.CustomMatcherFactories): void;
+        /**
+         * Add a custom reporter to the Jasmine environment.
+         */
         addReporter(reporter: jasmine.CustomReporter): void;
+        /**
+         * Adds a spec file to the list that will be loaded when the suite is executed.
+         */
         addSpecFile(filePath: string): void;
         addSpecFiles(files: string[]): void;
         addHelperFiles(files: string[]): void;
         addRequires(files: string[]): void;
-        configureDefaultReporter(options: any, ...args: any[]): void;
+        /**
+         * Configure the default reporter.
+         */
+        configureDefaultReporter(options: jasmine.DefaultReporterOptions): void;
```

Comment:
I can't find this in https://jasmine.github.io/api/npm/edge/Jasmine.html, could you point me to it?

Reply:
It is not on the documentation site as a public property but it was already in the types so I figured it should probably be updated to be correct.

Reply:
so it's a bogus declaration, should not be there IMO:
https://github.com/jasmine/jasmine/issues/1266
(that is the closes thing I can find)

Reply:
OK, here it is https://jasmine.github.io/setup/nodejs.html#reporters

Reply:
Jasmine NPM is a weird case where you have to dig into the source code, Its documentation is kinda lacking, or if it's there, it's in the NodeJS specific file and not with everything else.

Reply:
agree, found this, updated your comment on options

Reply:
Yeah, the API docs for -npm literally didn't exist a week ago. What's there now is just a first pass. It's pretty likely that there are some things missing from the docs that ought to be public.